rust/Cargo.toml

```diff
@@ -13,5 +13,8 @@ tokio = { version = "1.28.1", features = ["sync", "rt-multi-thread", "macros"] }
 rand = "0.8.5"
 modsurfer-demangle = { git = "https://github.com/dylibso/modsurfer" }
 log = "0.4.18"
-serde = { version = "1", features = ["derive"] }
-serde_json = "1"
+serde = { version = "1.0.164", features = ["derive"] }
+serde_json = "1.0.96"
```

Comment:
were these specific versions necessary to do anything you have in the PR?

Reply:
to add some context, we are trying to relax the versions as much as possible so that they a maximally compatible with the version an end-user has in their existing project.

Reply:
no, i borked the merge conflict. relaxing it again

Reply:
good eye!